partner_street_number/views/res_partner.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_partner_form" model="ir.ui.view">
+            <field name="name">Add separate fields for street name and number</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_form"/>
+            <field name="arch" type="xml">
+ 
+               <xpath expr="/form/sheet//div/field[@name='street']"
+                       position="replace">
+                    <div>
+                        <field name="street_name" style="width: 80%%"
+                               placeholder="Street..."
```

Comment:
Thanks for the review! I'm sorry, I don't exactly understand what you are hinting at here, please clarify.